File: Code/cpp/reverse.cpp
#include<iostream>
using namespace std;
int main()
{
    int num, reverse = 0;

    cout << "Enter Number: ";
    cin >> num;

    while (num != 0)
    {
        reverse = reverse * 10 + num % 10;
        num /= 10;
    }
    cout << reverse;
    return 0;
}

File: Code/java/reverse.java
import java.util.Scanner;

class Reverse
{
    public static void main(String[] args)
    {
        int num, reverse = 0;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Number: ");
        num = scanner.nextInt();

        while (num != 0)
        {
            reverse = reverse * 10 + num % 10;
            num /= 10;
        }
        System.out.println(reverse);

        scanner.close();
    }
}